src/main/java/Bosco.java

```diff
@@ -0,0 +1,96 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Bosco {
+    private static final String DIVIDER = "\t____________________________________________________________";
+    private static final int MAX_TASKS = 100;
+
+    private static final Task[] tasks = new Task[MAX_TASKS];
+    private static int taskCount = 0;
+
+    private static void printWelcomeMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Hello! I'm Bosco APD." + System.lineSeparator() + "\t What can I do for you?");
+        System.out.println(DIVIDER);
+    }
+
+    private static void printExitMessage() {
+        System.out.println(DIVIDER);
+        System.out.println("\t Bye! Hope to see you again soon!");
+        System.out.println(DIVIDER);
+    }
```

Comment:
I like how you abstracted the welcome and exit messages as methods, and grouped them together as they are related :)